tox.ini

```diff
@@ -17,11 +19,15 @@ envlist =
 
     py34-django17,
     py34-django18,
+    py34-django19,
+    py34-django110,
```

Comment:
Since we're updating tox, how about we also:

- [ ] Drop python 3.2 and 3.3.
- [ ] Drop Django 1.7.